lookout/style/format/features.py

```diff
@@ -171,26 +160,81 @@ def to_comment(self, correct_y: int) -> Comment:
                        CLASS_INDEX[CLS_NOOP]])
 
 
+FeatureId = Enum("FeatureId", "diff_col diff_line diff_offset index_internal_type index_label "
+                 "index_reserved indices_internal_type indices_label indices_reserved "
+                 "indices_roles length start_col start_line")
```

Comment:
I think it's actually good to be able to predict feature names easily. I'd rather have a separate function to make feature names readable.

Reply:
These names confuse me because it is not about indices. It is about value. Yes, it is encoded, but it is internal stuff. 

not so important. You decide. :)

Reply:
Feature names are internal stuff. I'll re-prefix with `simple` `ohe_single` and `ohe_multi` because we do have to distinguish between no ohe and ohe.

Reply:
oh. Maybe you misunderstand me. I mean having `index_internal_type` and `internal_type` features instead of  `index_internal_type` and `indices_internal_type`. definitely not `internal_type` and `internal_type`  :))

Reply:
ok, let's go for that.